chapters/joomla/index.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: default
+project: a
+title: Joomla
+chapter: true
+---
+
+<p align="center">
+  <img  src="http://woensdag.nl/wp-content/uploads/Joomla-website.png" width="70%"/>
+</br></br>
+
+<b>Remy Kabel, Panagiotis Mitsis, Michail Papadimitriou, Nikolaos Stekas</b><br/>
+<i>Delft University of Technology</i>
+</p>
```

Comment:
The a should indeed be Joomla, dunno how that happened. Ah, no, my bad on that link. (couldn't get the checker to work). The HTML is because I wanted the title image centered but then also the names.



> Op 30 apr. 2015 om 08:58 heeft Rick Wieman <notifications@github.com> het volgende geschreven:
> 
> In chapters/joomla/index.md:
> 
> > @@ -0,0 +1,253 @@
> > +---
> > +layout: default
> > +project: a
> > +title: Joomla
> > +chapter: true
> > +---
> > +
> > +<p align="center">
> > +  <img  src="http://woensdag.nl/wp-content/uploads/Joomla-website.png" width="70%"/>
> > +</br></br>
> > +
> > +<b>Remy Kabel, Panagiotis Mitsis, Michail Papadimitriou, Nikolaos Stekas</b><br/>
> > +<i>Delft University of Technology</i>
> > +</p>
> Why all the HTML here?
> 
> —
> Reply to this email directly or view it on GitHub.
>